features/xforms.xml

```diff
@@ -1,110 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
https://github.com/php/doc-en/pull/3601 Not yet deleted features/xforms.xml file

Reply:
The reason for going ahead and deleting `features/xforms.xml` now, in this and other translations, is so that when reference to it removed from `doc-base/manual.xml.in` it won't immediately break the build for all of those translations. In the (hopefully short) time between it being removed from the translations and from `doc-en`, the translations should just fall back to the English version.

Reply:
Updated when doc-en is deleted